docs/guides/qiskit-function-templates.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Qiskit Function templates
+description: Learn how to use and build Qiskit Function templates.
+---
+
+
+# Introduction to Qiskit Function templates
+
+[Qiskit Function templates](https://github.com/qiskit-community/qiskit-function-templates) are an open-source collection of realistic code examples that give you access to some of the latest research workflows that leverage tools like the Qiskit SDK and [Qiskit addons](https://quantum.cloud.ibm.com/docs/en/guides/addons). These pre-built workflows take learnings from practical experiments and package them in a way that you can customize and reuse in your own research so that you don't have to build from scratch. All templates are readily deployable to [Qiskit Serverless](./serverless) where you can easily leverage classical compute as part of running your workflow.
```

Comment:
```suggestion
[Qiskit Function templates](https://github.com/qiskit-community/qiskit-function-templates) are an open-source collection of realistic code examples that give you access to some of the latest research workflows that leverage tools like the Qiskit SDK and [Qiskit addons](https://quantum.cloud.ibm.com/docs/en/guides/addons). These pre-built workflows take information from practical experiments and package them in a way that you can customize and reuse in your own research so that you don't have to build from scratch. All templates are readily deployable to [Qiskit Serverless](./serverless) where you can easily leverage classical compute as part of your workflow.
```